web/src/views/Index.vue

```diff
@@ -221,7 +235,38 @@ export default defineComponent({
         html: true,
       }).onOk(() => {
         indexService.delete(props.row.name).then(() => {
-          getIndexes();
+          nextTick(getIndexes);
+        });
+      });
+    };
+
+    const deleteSelectedIndexes = () => {
+      if (!selectedIndexes || selectedIndexes.value.length == 0) {
+        $q.notify({
+          position: "top",
+          color: "warning",
+          textColor: "white",
+          icon: "warning",
+          message: "Please select index for delete",
```

Comment:
request you to change it to "Please select index for deletion"

Reply:
ok，thank